src/components/filters/Filters.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import FilterSelect from "./FilterSelect";
+import FilterRange from "./FilterRange";
+import FilterInput from "./FilterInput";
+import {FILTER_TYPE} from "../../constants";
+
+
+const Filters = ({
+  filterObject,
+  filters,
+  onChangeFilter,
+}) => {
```

Comment:
`Object` in `filterObject` isn't good, because:
 - doesn't bring any more information other than type
 - both `filterObject` and `filters` are objects, and adding Object might imply filters isn't an object

Better options:
 - `descriptions` and `values`?
 - `descriptionsByKey` and `valuesByKey`?